haproxy/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 2.3.5
 description: A Helm chart for HAProxy which can be customized by a config map.
 name: haproxy
-version: 1.3.7
+version: 1.3.8
```

I'd say this is a new backwards-compatible feature, justifying a minor version bump
```suggestion
version: 1.4.0
```